api/web3server_marshal.go

```diff
@@ -221,14 +222,64 @@ func (obj *getBlockResult) MarshalJSON() ([]byte, error) {
 	})
 }
 
+func newGetTransactionResult(
```

Comment:
the funcs in this file are to implement `MarshalJSON()` interface

Reply:
move to web3server_utils.go?